src/index.html

```diff
@@ -31,14 +30,14 @@ <h1 class="hero-header">Discover Amazing places in Japan</h1>
             <div class="search-bar">
                 <div class="search-question">
                     <label for="question1" class="question">
-                        <img class="oval-icon" src="/src/assets/icons/Oval.svg" alt="oval icon">
+                        <img class="oval-icon" src="/assets/icons/Oval.svg" alt="oval icon">
```

Comment:
naming image should be lowercase check and update for all

Reply:
I will check and update for all, thanks